src/screens/SUE/SueDetailScreen.tsx

```diff
@@ -142,7 +142,6 @@ export const SueDetailScreen = ({ navigation, route }: StackScreenProps<any>) =>
               <Map
                 isMaximizeButtonVisible
                 locations={[{ position: { latitude, longitude } }]}
-                mapStyle={styles.map}
                 onMaximizeButtonPress={() =>
                   navigation.navigate(ScreenName.MapView, {
                     locations: [{ position: { latitude, longitude } }]
```

Comment:
there are overlaying "buttons" now, which needs to be avoided

<img width="381" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/42b08217-0dfc-44ab-9021-4fa25d46a212">

Reply:
should we place the "maximize map button" in the top right corner instead the bottom right one?

Reply:
should we use it like this?
<image src="https://github.com/user-attachments/assets/b2f39106-69d0-4018-92bc-e90109c51edf" width="250" />

ur put it in the bottom left corner with the old styling? dosen't seem like we use the bottom left corner for anything.
<image src="https://github.com/user-attachments/assets/6993d10f-e53d-4853-929b-b609c8ef082a" width="250" />

Reply:
first! some actions right others left is no option. additionally positioning left is bad ux for most users which are right handed

Reply:
applied the fist change.